.env

```diff
@@ -1,3 +1,4 @@
 VUE_APP_API_ENDPOINT={Endpoint}
 ROOT=http://localhost:8080/
-TITLE=techKnow (Dev)
\ No newline at end of file
+TITLE=techKnow (Dev)
+VUE_APP_GOOGLE_ANALYTICS_ID=UA-180455475-1
```

Comment:
Should this be a variable that can be changed on deployment rather than hardcoded?

Reply:
Thanks, I've made this into a variable